css-text-3/Overview.bs

```diff
@@ -1549,7 +1549,7 @@ Text Processing</h3>
 
     <p>As each line is laid out,</p>
     <ol>
-      <li>A sequence of collapsible spaces at the beginning of a line
+      <li>A sequence of [=collapsible=] [=white space=] at the beginning of a line
```

Comment:
This should stay spaces as well, since we've converted collapsible white space to spaces.

Reply:
Aren't **collapsible** line breaks removed (because they're turned into spaces), and it's only preserved non collapsible line breaks that say?

Reply:
Sure, but now that they're spaces they should just be called spaces.